upstash_vector/core/index_operations.py

```diff
@@ -100,17 +121,38 @@ def query(
             include_metadata=True,
         )
         ```
+
+        ```python
+        query_res = index.query(
+            data="hello"
+            top_k=3,
+            include_vectors=True,
+            include_metadata=True,
+        )
+        ```
         """
         payload = {
-            "vector": convert_to_list(vector),
             "topK": top_k,
             "includeVectors": include_vectors,
             "includeMetadata": include_metadata,
             "filter": filter,
         }
+
+        if data is None and vector is None:
+            raise ClientError("either `data` or `vector` values must be given")
+        if data is not None and vector is not None:
+            raise ClientError("data and vector values cannot be given at the same time")
```

Comment:
```suggestion
            raise ClientError("`data` and `vector` values cannot be given at the same time")
```
minor: just for consistency